Makefile

```
@@ -43,6 +69,18 @@ endif
 endif
 	@# This comment silences warning "make: Nothing to be done for 'swag'."
 
-deps:
-	cd .. && go get -u github.com/swaggo/swag/cmd/swag@v1.7.1
-	go mod download
+lint: lint-md lint-go
+lint-fix: lint-fix-md
```

Does not contain `lint-fix-go`
```suggestion
lint-fix: lint-fix-md lint-fix-go
```

---

Got damn it x)